OpenRA.Game/Map/Map.cs

```diff
@@ -695,16 +695,16 @@ public byte[] SavePreview()
 							var dx = uv.V & 1;
 							if (x + dx > 0)
 							{
-								var z = y * stride + 8 * x + dx - 4;
+								var z = y * stride + 8 * x + 4 * dx - 4;
```

Comment:
This would be clearer as
```
var pxStride = 4; // Defined somewhere much further up
var z = y * stride + pxStride * (2 * x + dx - 1);
```
and then `pxStride` used in the other places too.